bsip-0003.md

```diff
@@ -2,7 +2,7 @@
     Title: Maker / Taker Market Fees Flag
     Authors: Daniel Larimer <Dan@cryptonomex.com>
              Fabian Schuh <Fabian@BitShares.eu>
-    Status: Deferred
+    Status: Obsoleted
```

Comment:
Suggest to mark it as "Obsoleted by <BSIP>" and include a reference to the current BSIP/discussion.

Reply:
There is one.